autoload/neomake/makers/ft/haskell.vim

```diff
@@ -1,10 +1,23 @@
 
 function! neomake#makers#ft#haskell#EnabledMakers()
-    return ['ghcmod', 'hdevtools', 'hlint']
+    let makers = []
+    if executable('ghc-mod')   | call add(makers, 'ghcmod')    | endif
+    if executable('hdevtools') | call add(makers, 'hdevtools') | endif
+    if executable('hlint')     | call add(makers, 'hlint')     | endif
+    if executable('liquid')    | call add(makers, 'liquid')    | endif
+    return makers
```

Comment:
Just return a list here, the `executable` check gets done by Neomake then.
It makes sense though to check it yourself, is some meta-checker should be preferred over others and/or two makers are basically the same etc.